svc/pkg/job-log/ops/read/src/lib.rs

```diff
@@ -3,6 +3,7 @@ use rivet_operation::prelude::*;
 
 #[derive(clickhouse::Row, serde::Deserialize)]
 struct LogEntry {
+	// In nanoseconds
 	ts: i64,
```

Comment:
Should this `ts` variable name be changed to match others changing to `nts`?

Reply:
This is parsing from the database, i chose to leave it as `ts`